bitnami/concourse/templates/worker/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.worker.enabled -}}
+{{- if .Values.worker.podSecurityPolicy.enabled -}}
```

Comment:
For consistency with other charts, change podSecurityPolicy.enabled to psp.create. This is what we have in most of the charts